sos/sosreport.py

```diff
@@ -1609,7 +1609,7 @@ def execute(self):
                     rmtree(self.tmpdir)
             except:
                 raise
-
+            raise to_be_rasied
```

Comment:
There is spelling mistake, it should be 'to_be_raised' instead of 'to_be_rasied'